install_config/install/disconnected_install.adoc

```diff
@@ -489,6 +489,46 @@ registry.access.redhat.com/jboss-webserver-3/webserver30-tomcat7-openshift:lates
 $REGISTRY:5000/openshift/webserver30-tomcat7-openshift:latest
 ----
 
+[[diconnected-configuring-registry-location]]
+=== Configuring a Registry Location
+
+If you are using an image registry other than the default at
+`registry.access.redhat.com`, specify the desired registry within the
+*_/etc/ansible/hosts_* file.
+
+----
+oreg_url=example.com/openshift3/ose-${component}:${version}
+openshift_examples_modify_imagestreams=true
+----
+
+If your registry is secure, you will also need to configure:
+----
+openshift_docker_additional_registries=$REGISTRY:5000
+openshift_docker_insecure_registries=$REGISTRY:5000
+----
+
+.Registry Variables
+[options="header"]
+|===
+
+|Variable |Purpose
+|`*oreg_url*`
+|Set to the alternate image location. Necessary if you are not using the default
+registry at `registry.access.redhat.com`.
+
+|`*openshift_examples_modify_imagestreams*`
+|Set to `true` if pointing to a registry other than the default. Modifies the
+image stream location to the value of `*oreg_url*`.
+
+|`*openshift_docker_additional_registries*`
+|Set to the URL created with the docker images. Verify that the name and label of
+the repository match the original name.
+
+|`*openshift_docker_insecure_registries*`
+|Set to the URL created with the docker images. Verify that the name and label of
+the repository match the original name.
+|===
+
```

Comment:
I do not think the explanation for openshift_docker_additional_registries and openshift_docker_insecure_registries are correct.